### ISSUE TITLE
Design Suggestions:  Font Styles and Collapsed menus by default.

### ISSUE THREAD

**Issue by icnikerazah** [godotengine/godot#14380]
**Issue description:**

Godot gui is illegible. Very flat and monochrome design is bad for eyes.
1. All tree-view menus should be collapsed by default.
2. Sections and Propertions titles are unneccessary.
2. Font styles must be applied.

Example Design
**BEFORE**
![before1](https://user-images.githubusercontent.com/33985414/33720837-4409ded4-db76-11e7-83c5-2c8622bec4bb.png)
**AFTER**
![after2](https://user-images.githubusercontent.com/33985414/33720858-523074f0-db76-11e7-8790-18873f89dd41.png)

**Comment by MrMaidx** [comment]
My opinion: I'm all for bold font for titles and removal of Sections and Properties titles because it's pretty obvious what is what. I don't like cursive in tree view and I'm indifferent about collapsing.

**Comment by icnikerazah** [comment]
it's not cursivec. its default font Noto italic

**Comment by Zireael07** [comment]
cursive = italic

**Comment by icnikerazah** [comment]
i am graphic designer. sans serif italic style is not cursive.

**Comment by icnikerazah** [comment]
titles and sub sections can't be same font weight and style. sorry for my english

**Comment by akien-mga** [comment]
CC @djrm @toger5 @volzhs

**Comment by Zireael07** [comment]
@icnikerazah: Many people here are not native English speakers. I always equated italic with cursive too and I'm not surprised other people do.

**Comment by leoddd** [comment]
Love everything except for the fully capitalized tabs.
Makes it harder to quickly differentiate between them at a glance, since they now all have roughly the same silhouette.

SEARCH is fine if that is globally capitalized.

**Comment by capnm** [comment]
@Zireael07 Yes, at least in Europe italic type is a cursive font and vice versa
 https://en.wikipedia.org/wiki/Italic_type
The point is: I also think that it looks ugly and it should be the same font, with the same size, but not the italic style.

**Comment by icnikerazah** [comment]
capnm. it looks ugly, i agree. This example design only a prototype.  but should not be the same as the left panel the right panel font weights.

**Comment by djrm** [comment]
bold fonts make sense, the rest not much, at some point i wanted to remove panel titles, but after revieweing most of the interface i decided not to do so because i could not remove it everywhere, which would make those placed look weird, so i decided to leave them.

having said that, you have enough content differentiation by their hierarchichal positioning (could be improved by making it stand out even more as you point out, but is not confusing either)

also i really dont feel like working on this, so if anyone else want to tackle it go ahead, sorry.

**Comment by toger5** [comment]
Bold would be great. That was mentioned in a couple of discussions by other ppl to.
especially for the active tabs

**Comment by volzhs** [comment]
I love to have blod/italic text without actual font files for it.
I found this long time ago.
https://lists.nongnu.org/archive/html/freetype/2002-09/msg00081.html
would someone implement and test it?

**Comment by Calinou** [comment]
> I love to have blod/italic text without actual font files for it.

Noto Sans *does* have bold weights available, you'll just need to add them to the Godot repository. 😄

**Comment by icnikerazah** [comment]
All Noto Sans font family can be used. for example Condensed for tabs, semi consended for properties  etc

**Comment by volzhs** [comment]
@Calinou I know there is bold font for Noto sans.
but I think there is a way to make it looks bold or italic without additional font files.

**Comment by capnm** [comment]
@volzhs That's a very bad idea. The font to make so broken is meant only as an absolute last resort.

**Comment by RayKoopa** [comment]
The italic is total nonsense and doesn't add anything to the UX. The sections are already separated. There's nothing else non-italic in the left section so it's useless.
Bold however adds differentation between category titles and actual items, so I'd say go for it.

(I'm a UI developer / designer and UX engineer professionally. Does that make my comment weigh more? 🗡 )

**Comment by icnikerazah** [comment]
looks like Animation Player in Animation Player. What is in what? this design sucks
<img width="304" alt="ekran resmi 2017-12-13 23 06 28" src="https://user-images.githubusercontent.com/33985414/33960026-6b56c536-e05a-11e7-9f7f-d93ed582f5e4.png">

**Comment by leoddd** [comment]
Nothing is in anything. The inspector displays which class the properties below belong to.
Everything down to the Node heading is part of AnimationPlayer's class, while everything under Node is a part of the Node class, which AnimationPlayer inherits.

If that node were one based on AnimationPlayer, it would show a different name at the top.

**Comment by RayKoopa** [comment]
Maybe if you would've actually used Godot before trying to apply design to it, you would've realized the top "AnimationPlayer" text is the node name, which just happens to be the same as the class name for new nodes, so it is "repeated" in the inspector like @leoddd explained.

**Comment by icnikerazah** [comment]
I already know what you're talking about. Node name already available in Scene tab. Node name in node name is a bad idea

**Comment by icnikerazah** [comment]
Ray koopa : bla in bla ? fucking Recursive panel design,

**Comment by RayKoopa** [comment]
Sorry, but you don't know what you're talking about. The tool windows can be separated and the Scene tab might not even be visible at the same time. Every property editor of an IDE lists the currently edited object name somewhere.

**Comment by icnikerazah** [comment]
Okey.
Here is another oddity;
1. Add a Node and select it in Scene tab.
2. Double click icon.png in FileSystem ( png is in Inspector)
3. Select Node.
Whola! you could choose the already selected one.

![1-appbwh6_rtvocvxwqpplha](https://user-images.githubusercontent.com/33985414/33961511-6612d8a8-e05f-11e7-9d99-af0664ec9d59.jpeg)

**Comment by RayKoopa** [comment]
Which again just proves my point that an inspector requires to display the object it currently edits. Thanks.

**Comment by icnikerazah** [comment]
How can two things be chosen at the same time? If I am editing png, why is the node selected.

**Comment by RayKoopa** [comment]
Because it is still active in the Scene. You still see transformation overlays in the scene view.

**Comment by akien-mga** [comment]
@icnikerazah The saltiness is not appreciated. Be polite or constructive, or begone. Thanks.

**Comment by icnikerazah** [comment]
You are right. I apologize. I'm a little nervous. I will not write this title.
